modules/fedimint-lnv2-client/src/lib.rs

```diff
@@ -515,8 +515,20 @@ impl LightningClientModule {
             .await
     }
 
-    /// Pay an invoice. For  testing  you can optionally specify a gateway to
-    /// route with, otherwise a gateway will be selected automatically.
+    /// Pay an invoice. For testing you can optionally specify a gateway to
```

Comment:
Please change "If the invoice was created by a gateway connected to our federation this gateway will be selected, otherwise we select a random online gateway."

to:

"If the invoice was created by a gateway connected to our federation, the same gateway will be selected to allow for a direct ecash swap. Otherwise we select a random online gateway."

Reply:
Done.